app/services/stats/compute_rate_of_applicants_with_rdv_seen.rb

```diff
@@ -0,0 +1,31 @@
+module Stats
+  class ComputeRateOfApplicantsWithRdvSeen < BaseService
+    def initialize(rdv_contexts:)
```

Comment:
si ce service est censé faire la même chose que l'autre service, pourquoi avoir 2 services différents ? 
Pourquoi pour un mois donnée on ne prendrait pas en considération uniquement les `rdv_contexts` créé dans le mois en question ?

Reply:
Parce qu'on peut créer des `rdv_contexts` sans envoyer d'invitation ou sans convoquer. Ces bénéficiaires ne doivent pas être pris en compte dans le mois par mois.

Reply:
D'où ma remarque au-dessus ☝️ . 
Si on ne garde que ce service, dans lequel on ne garderait que les `rdv_contexts` qui ont des invitations et des convocations, on pourrait avoir le mois par mois via la date de création du contexte et qui correspondrait bien entre le global et le local. 
Du coup on la formule serait 
`Pour le moins M, on a ouvert un suivi pour N usagers et sur ces N usagers il y en a K qui ont été orienté in fine via l'appli`

Reply:
je ne suis pas d'accord, ça ne nous permettrait pas de savoir quand des bénéficiaires sont re-rentrés dans le dispositif, et de voir si ils ont bien été orienté aussi cette seconde fois. D'où le trade-off proposé par Claire, qui n'est pas idéal mais est plus fiable selon moi étant donné que les bénéficiaires sont rarement re-invités/convoqués ou bout de quelques jours/semaines si le rendez-vous n'est pas pris

Reply:
Ah mais je pensais qu'on voulait justement ne pas prendre en compte les ré-orientés. 
Mais pour les ré-orientés t'auras le problème inverse sur l'autre service: si l'usager apparait N fois dans la liste des personnes à inviter, et que les N-1 premières fois il ne prend pas rdv et qu'il prend rdv la N-ième fois, on va considérer qu'il a été orienté N fois. Pour moi c'est pire. 
Je me disais qu'on voulait justement pas prendre en compte les ré-orientés (qui restent marginaux pour l'instant) pour avoir un calcul plus juste.

Reply:
on va aller au plus simple pour l'instant et privilégier ce que tu proposes dans un premier temps